aptible/endpoints.go

```diff
@@ -63,7 +66,10 @@ func (c *Client) CreateEndpoint(service Service, attrs EndpointCreateAttrs) (End
 
 	if *attrs.Type != "tcp" {
```

Comment:
I think this should actually be of the logic `not TCP and not TLS` 

or since our accepted options are just HTTPs/TCP/TLS, just make it

if `attrs.Type == "https"

Reply:
so only https supports singular ports i guess, can do this

Reply:
ex on public repo (https://github.com/aptible/aptible-cli/blob/2baa61e9ca55224d659d784fb4a8b14a3b7dbbb1/lib/aptible/cli/subcommands/endpoints.rb#L120) sharing as this is public as well (can't really share internal refs here):

```rb
              create_app_vhost(
                https_create_flags, options, type,
                type: 'http', platform: 'alb'
              )
```

Reply:
it turns out it's `http_proxy_protocol` (not https), but updated accordingly